### PR TITLE
Stylised Checkbox

### LINKED ISSUE #2: Stylised checkbox
Currently a standard checkbox is used instead of the stylised checkbox in the actual WP login screen.

WordPress seems to have some pretty standard styles for the _unchecked_ checkbox, which could be directly copied, but then it does something interesting with pseudo-selectors for the _checked_ version:

```
input[type=checkbox]:checked:before {
    font: normal 21px/1 dashicons;
    content: "\f147";
    margin: -3px 0 0 -4px;
    color: #1e8cbe;
    float: left;
    display: inline-block;
    vertical-align: middle;
    width: 16px;
    speak: none;
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
}
```

Looks like the selector targets the "checked" version specifically with the `:checked` psuedo-selector, and then uses `:before` to add a child (more info [here](https://developer.mozilla.org/en-US/docs/Web/CSS/::before)).

The child receives a custom font called [Dashicons](https://developer.wordpress.org/resource/dashicons/#yes) and the content is set to a strange character which must be the equivalent of a check in that font.

### PR DESCRIPTION
**Pull request by JordanCarbis**
Hey just finished those checkbox issues you pointed out to me. If you could have a look over the changes that would be great thanks!

Closes #2

### CODE REVIEW

**Comment by lukecarbis** [comment]
Hey! I'm getting a weird display thing when I click the checkbox.

Screen Shot 2019-03-14 at 12 33 50 pm

I wonder if it's related to the console error I'm getting too?

Screen Shot 2019-03-14 at 12 35 13 pm

File: style.css [hunk 1]
```diff
@@ -157,3 +157,39 @@ body {
...
+    display: inline-block;
+    vertical-align: middle;
+    width: 16px;
+    speak: none;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+}
+@font-face {
+    font-family:"dashicons";
+    src: url(/fonts/dashicons.ttf)
```
Comment (lukecarbis) [inline style.css]: Should there be a closing `}` tag here?

**Review by lukecarbis — CHANGES_REQUESTED** [review]
Code looks good. Having some problems (see previous comments), and there's one minor change suggested in this review too.

File: style.css (hunk 1 above)
Comment (JordanCarbis) [reply style.css]: Display error and minor change to code has been fixed in c54586e

**Review by lukecarbis — APPROVED** [review]

**Pull request merged**